cfme/tests/infrastructure/test_host.py

```diff
@@ -529,6 +536,44 @@ def test_add_ipmi_refresh(appliance, setup_provider):
     assert view.ipmi_address.read() == ipmi_address
 
 
+@pytest.fixture
+def host(appliance, provider):
+    host_collection = provider.hosts
+    return random.choice(host_collection.all())
+
+
+HOST_DETAIL_ROWS = [
+    ("Properties", "VMM Information", HostVmmInfoView),
+    ("Properties", "Operating System", HostOsView),
+    ("Properties", "Devices", HostDevicesView),
+    ("Properties", "Networks", HostNetworkDetailsView),
+    ("Properties", "Storage Adapters", HostStorageAdaptersView),
+    ("Configuration", "Services", HostServicesView),
+    ("Toolbar", "Print or export", HostPrintView),
+]
+
+
+@test_requirements.infra_hosts
+@pytest.mark.parametrize("table,row,view", HOST_DETAIL_ROWS,
+    ids=[rel[1] for rel in HOST_DETAIL_ROWS])
+def test_infrastructure_hosts_viewing(appliance, setup_provider, host, table, row, view):
```

Comment:
What is the purpose of the `table` parameter here? The navigation destinations select this automatically, I think it only would have been necessary if you were interacting with the summary table widgets directly within the test.

Reply:
Yep. I forgot to remove when I created the navigation. Removing.

Reply:
But wait. I used this to check that the value was not '0'. Might need to keep it.